.github/workflows/installer.yml

```diff
@@ -0,0 +1,57 @@
+name: Installer integration
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "installer.sh"
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  ubuntu:
+    name: Test installer on Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Install code-server
+        run: ./install.sh
+
+      - name: Test code-server
+        run: yarn test:standalone-release code-server
+
+  alpine:
+    name: Test installer on Alpine
+    runs-on: ubuntu-latest
+    container: "alpine:3.14"
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Install curl
+        run: apk add curl
+
+      - name: Add user
+        run: adduser coder --disabled-password
+
+      # Standalone should work without root.
+      - name: Test standalone to a non-existent prefix
+        run: su coder -c "./install.sh --method standalone --prefix /tmp/does/not/yet/exist"
```

Comment:
TIL that `su` makes you a superuser? (according to explainshell)

Reply:
My guess is that this is running as root by default, so `su coder` means "run as coder user" and the `-c` defines the shell command to run. This explanation looks decent: https://unix.stackexchange.com/questions/257698/how-are-setuid-suid-sudo-and-su-all-related

Reply:
> The most common use of su is to become root

oh! that makes more sense now. thanks for the link and explanation!